automata/symbol/graph/symbol_graph.py

```diff
@@ -40,6 +46,10 @@ def __init__(
         self._graph = builder.build_graph()
         self.navigator = SymbolGraphNavigator(self._graph)
 
+        if pickle_graph:
+            with open(f"{data_root_path}/symbol_graph.pkl", "wb") as f:
```

Comment:
FYI you are calling this correctly here, but you are naming it `symbolgraph.pkl` in other places, you might want to check everything is consistent.